src/gluonnlp/model/bert.py

```diff
@@ -619,6 +632,10 @@ def get_bert_model(model_name=None, dataset_name=None, vocab=None,
         Whether to include the decoder for masked language model prediction.
     use_classifier : bool, default True
         Whether to include the classifier for next sentence classification.
+    output_attention : bool, default False
+        Whether to include attention weights of each encoding cell to the output.
+    output_all_encodings : bool, default False
+        Whether to include intermediate encodings of each encoding cell to the output.
```

Comment:
Whether to  to output encodings of all encoder cells

Reply:
Fixed